checker/src/main/java/org/checkerframework/checker/mustcall/MustCallVisitor.java

```diff
@@ -59,6 +64,39 @@ public Void visitReturn(ReturnTree node, Void p) {
     return super.visitReturn(node, p);
   }
 
+  @Override
+  protected boolean validateType(Tree tree, AnnotatedTypeMirror type) {
+    if (TreeUtils.isClassTree(tree)) {
+      Element classEle = TreeUtils.elementFromDeclaration((ClassTree) tree);
+      AnnotationMirror inheritableMustCall =
+          atypeFactory.getDeclAnnotation(classEle, InheritableMustCall.class);
+      if (inheritableMustCall != null) {
+        AnnotationMirror explict = atypeFactory.fromElement(classEle).getAnnotation();
+        if (explict != null) {
+          List<String> mustCallVal =
+              AnnotationUtils.getElementValueArray(
+                  inheritableMustCall, atypeFactory.inheritableMustCallValueElement, String.class);
+          AnnotationMirror inheritedMCAnno = atypeFactory.createMustCall(mustCallVal);
+
+          // Issue an error if there is an inconsistent, user-written @MustCall annotation.
+          AnnotationMirror writtenMCAnno = type.getAnnotation();
+          if (writtenMCAnno != null
+              && !atypeFactory.getQualifierHierarchy().isSubtype(inheritedMCAnno, writtenMCAnno)) {
+
+            checker.reportError(
```

Comment:
This will only issue a warning for classes in source code.  With out this PR, the error was issued for any use of a class.  Is that ok?

Reply:
Yes. I think that's actually preferable, because it obviates the need for `issuedInconsistentMustCallSubtype` (which was really a kludge that didn't work all that well).